e2e/plugin/plugin_test.go

```diff
@@ -123,6 +124,23 @@ var _ = Describe("Test Kubectl Plugin", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(output).Should(ContainSubstring("Properties"))
 		})
+		It("Test show componentDefinition def with raw Kube mode", func() {
+			cdName := "kube-worker"
+			output, err := e2e.Exec(fmt.Sprintf("kubectl-vela show %s", cdName))
+			Expect(err).NotTo(HaveOccurred())
+			arr := strings.Split(output, "\n")
+			k1 := strings.Split(arr[4], "|")[1]
+			v1 := strings.Split(arr[4], "|")[2]
+			k2 := strings.Split(arr[5], "|")[1]
+			v2 := strings.Split(arr[5], "|")[2]
+			paraMap := make(map[string]string)
+			paraMap[k1] = v1
+			paraMap[k2] = v2
+			Expect(paraMap).Should(Equal(map[string]string{
+				" image ": " The value will be applied to fields: [spec.template.spec.containers[0].image].                  ",
+				" port  ": " The value will be applied to fields: [spec.template.spec.containers[0].ports[0].containerPort]. ",
```

Comment:
Let's use the description field 

![image](https://user-images.githubusercontent.com/2173670/119249723-6741fe00-bbcd-11eb-9cd4-1d795d7e0f12.png)